man/dockerd.8.md

```diff
@@ -78,6 +80,12 @@ format.
 **--api-cors-header**=""
   Set CORS headers in the remote API. Default is cors disabled. Give urls like "http://foo, http://bar, ...". Give "*" to allow all.
 
+**--authn**=*true*|*false*
```

Comment:
I don't see why this option is necessary. Surely this should just be based on whether or not any `authn` plugins were loaded.